queries.md

```diff
@@ -4,76 +4,94 @@
 
 ### 1. All the companies whose name match 'Babelgum'. Retrieve only their `name` field.
 
-<!-- Your Code Goes Here -->
+- Query: {name: "Babelgum"}
+- Projection: {name:1, _id:0}
 
 ### 2. All the companies that have more than 5000 employees. Limit the search to 20 companies and sort them by **number of employees**.
 
-<!-- Your Code Goes Here -->
+- Query: {number_of_employees:  {$gt: 5000}}
+- Sort: {number_of_employees:1}
+- Limit: 20
 
 ### 3. All the companies founded between 2000 and 2005, both years included. Retrieve only the `name` and `founded_year` fields.
 
-<!-- Your Code Goes Here -->
+- Query: {$and: [{founded_year: {$gte:2000}}, {founded_year: {$lte: 2005}}]}
+- Projection: {name: 1, founded_year:1}
 
 ### 4. All the companies that had a Valuation Amount of more than 100.000.000 and have been founded before 2010. Retrieve only the `name` and `ipo` fields.
 
-<!-- Your Code Goes Here -->
+- Query: {$and: [{"ipo.valuation_amount":{$gt:100000000}}, {founded_year: {$lt: 2005}}]}
+- Projection: {name:1, ipo:1}
 
 ### 5. All the companies that have less than 1000 employees and have been founded before 2005. Order them by the number of employees and limit the search to 10 companies.
 
-<!-- Your Code Goes Here -->
+- Query: {$and: [{number_of_employees: {$lt:1000}}, {founded_year: {$lt:2005}}]}
+- Sort: {number_of_employees:1}
+- Limit: 10
 
 ### 6. All the companies that don't include the `partners` field.
 
-<!-- Your Code Goes Here -->
+- Query: {partners:{$exists:false}}
 
 ### 7. All the companies that have a null type of value on the `category_code` field.
 
-<!-- Your Code Goes Here -->
+- Query: {category_code:null}
 
 ### 8. All the companies that have at least 100 employees but less than 1000. Retrieve only the `name` and `number of employees` fields.
 
-<!-- Your Code Goes Here -->
+- Query: {$and:[{number_of_employees: {$gt:100}}, {number_of_employees:{$lt:1000}}]}
```

Comment:
i think i have answered this one prevoious comment:
you can chain everything inside the same "object"

```js 
{number_of_employees: {  $gte: 100, $lt: 1000  }  }
```